config-gen/examples/advancedextension/README.md

```diff
@@ -0,0 +1,41 @@
+# Kustomize Example
+
+This directory contains an example of using `kubebuilder alpha config-gen` as a Kustomize transformer
+plugin.
+
+This enables using `config-gen` in traditional kustomize workflows (patch, bases, etc).
+
+## Setup `kubebuilder config-gen` as a plugin
+
+```
+export XDG_CONFIG_HOME=~/.config
+
+# setup the kubebuilder plugin
+export KUBEBUILDER_PLUGIN=$XDG_CONFIG_HOME/kustomize/plugin/kubebuilder.sigs.k8s.io/kubebuilderproject
+mkdir -p $KUBEBUILDER_PLUGIN
+cat > $KUBEBUILDER_PLUGIN/KubebuilderProject <<EOF
+#!/bin/bash 
```

Comment:
Also, not easy here to provide the examples only doc the config that should be passed to the kubebuilder alpha config-gen?

Reply:
https://github.com/kubernetes-sigs/kubebuilder/pull/1831/files#diff-7664f14e421dc7d2b17ee6081de4b2eb91be74aa6e667a6de143830768bb676e

Reply:
I'll clean this doc up a bit.